lib/xandra/options_validators.ex

```diff
@@ -40,6 +40,10 @@ defmodule Xandra.OptionsValidators do
     end
   end
 
+  def validate_node(%Xandra.Cluster.Host{address: address, port: port}) do
+    {:ok, {address, port}}
```

Comment:
For more context, `address` here might not strictly be a charlist, it could also be an `:inet.ip_address()`, effectively breaking the typespec of this `validate_node` function. But that is exactly what both `:ssl` and `:gen_tcp` expect, so connections should still established correctly.